src/components/formly.field.ts

```diff
@@ -66,6 +66,7 @@ export class FormlyField implements DoCheck, OnInit {
 
   ngOnInit() {
     this.createChildFields();
+    this.checkDefaultValue();
```

Comment:
I wanted field.key of nested.arrays.0, but in FormlyForm, I was only getting key as 0, so I decided to have it here in FormlyField